tests/test_modeling_speech_to_text.py

```diff
@@ -724,7 +724,7 @@ def map_to_array(batch):
             return batch
 
         ds = load_dataset("patrickvonplaten/librispeech_asr_dummy", "clean", split="validation")
-        ds = ds.select(range(num_samples)).map(map_to_array)
+        ds = ds.sort("id").select(range(num_samples)).map(map_to_array)
```

Comment:
_Unrelated to this PR_: the new`datasets` version re-shuffled this dataset, so sorting is needed for reproducibility.